src/common/telemetry/collect-telemetry/collect-telemetry.ts

```diff
@@ -0,0 +1,149 @@
+import {execSync} from 'child_process'
+import * as fs from 'fs'
+import fetch from 'node-fetch'
+import * as path from 'path'
+import {ProjenrcFile} from 'projen'
+import type {NodeProject, NodeProjectOptions} from 'projen/lib/javascript'
+import type {IWithTelemetryReportUrl} from '../with-telemetry'
+import {cloneWorkflow} from './clone-workflow'
+import {diff} from './diff'
+import {EscapeHatches, getEscapeHatches} from './get-escape-hatches'
+
+interface TelemetryPayload {
```

Comment:
I've configured the `consistent-type-definitions` rule to require `interfaces`, since JSII does not work with `type` for exported members.

This one is not exported and thus can be converted to `type`. However it would require either to disable the _eslint_ rule per line or completely switch it off for the repo.

Reply:
@quesabe do you mean that the aslant rule in the repo requires `interface`s instead of `type`s? If so, then I'm ok with interface. Just wanted it to stay consistent.

Reply:
Please resolve the conversation if what I'm saying above is correct.